src/components/Feed/Socialize/CommentLine.tsx

```diff
@@ -0,0 +1,85 @@
+import { useFragment } from 'react-relay';
```

Comment:
this file is for this component 
<img width="181" alt="image" src="https://user-images.githubusercontent.com/6754223/195633503-9784eb18-cef2-4c62-995f-ed7c96c26a7e.png">